components/Layout/Footer.tsx

```diff
@@ -7,7 +7,7 @@ import styles from './Footer.module.css'
 export const Footer = () => {
   const tsln = useTranslation<WebTranslations>()
   return (
-    <footer className="text-[14px]">
+    <footer id="wb-info" className="text-[14px]">
```

Comment:
this id could probably be changed to something more relevant, like `footer` (note to change the href in the layout index file too)